packages/frontend/components/scaffold-eth/Contract/DisplayVariables.tsx

```diff
@@ -0,0 +1,49 @@
+import { ContractFunction } from "ethers";
+import { FunctionFragment } from "ethers/lib/utils";
+import React, { FC, SetStateAction, useCallback, useEffect, useState, Dispatch } from "react";
+// import { useIsMounted } from "usehooks-ts";
+import { tryToDisplay } from "./displayUtils";
+
+interface IDisplayVariableProps {
+  contractFunction: ContractFunction | undefined;
+  functionInfo: FunctionFragment;
+  refreshRequired: boolean;
+  setTriggerRefresh: Dispatch<SetStateAction<boolean>>;
+}
+
+// TODO Check for the need of useIsMounted
```

Comment:
Is this because the hydration problem? In some case when you refresh the page, you get an hydration error (next server prerendering doesn't match the client)

I encounter this in some components (Like the address one) in case you want to take a look